<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_wael</class>
 <widget class="QMainWindow" name="gestion_wael">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/img/hand-painted-watercolor-background-with-sky-clouds-shape_24972-1095.jpg);
background-color: rgb(214, 0, 0);
background-image: url(:/new/prefix1/img/maxresdefault.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>901</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 0, 0);</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Equipements</string>
     </attribute>
     <widget class="QLineEdit" name="rech_equipement">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="table_equipement">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>71</y>
        <width>651</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#e54d30;&quot;&gt;Gestion Equipements&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="equipement_sel_col">
      <property name="geometry">
       <rect>
        <x>591</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Date Achat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prix</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Etat</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>UPLOAD IMAGE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>420</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>480</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PRIX TOTAL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>426</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>134</x>
        <y>475</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>UPLOAD VIDEO</string>
      </property>
     </widget>
     <widget class="QLabel" name="Clocklabel">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="date">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>45</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Maintenances</string>
     </attribute>
     <widget class="QTableView" name="table_maintenance">
      <property name="geometry">
       <rect>
        <x>-50</x>
        <y>0</y>
        <width>651</width>
        <height>341</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#e54d30;&quot;&gt;Gestion Maintenance&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rech_maintenance">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="maintenance_sel_col">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Equipement</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Duree</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Total</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Type Panne</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>420</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>450</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>EXCEL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="play">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>510</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>510</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionadd_equipement"/>
   <addaction name="actiondelete_equipement"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionadd_maintenance"/>
   <addaction name="actiondelete_maintenance"/>
  </widget>
  <action name="actionadd_equipement">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/img/images (2).jfif</normaloff>:/new/prefix1/img/images (2).jfif</iconset>
   </property>
   <property name="text">
    <string>add_equipement</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/prefix1/img/images (2).jfif&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actiondelete_equipement">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/img/images (1).jfif</normaloff>:/new/prefix1/img/images (1).jfif</iconset>
   </property>
   <property name="text">
    <string>delete_equipement</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/prefix1/img/images (1).jfif&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionadd_maintenance">
   <property name="text">
    <string>add_maintenance</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actiondelete_maintenance">
   <property name="text">
    <string>delete_maintenance</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
